tools/scenario_runner.py

```diff
@@ -1,26 +1,26 @@
-# -*- coding: utf8 -*-
+# -*- coding: utf-8 -*-
 from __future__ import print_function
-
-from gevent import monkey
```

Comment:
http://www.gevent.org/gevent.monkey.html

> Patching should be done as early as possible in the lifecycle of the program. For example, the main module (the one that tests against __main__ or is otherwise the first imported) should begin with this code, ideally before any other imports: